lib/src/main/kotlin/at/bitfire/ical4android/AndroidEvent.kt

```diff
@@ -132,20 +131,22 @@ abstract class AndroidEvent(
      * @param event event that can be saved into the calendar storage
      */
     constructor(calendar: AndroidCalendar<AndroidEvent>, event: Event) : this(calendar) {
-        this.event = event
+        this._event = event
     }
 
-    var event: Event? = null
-        /**
-         * This getter returns the full event data, either from [event] or, if [event] is null, by reading event
-         * number [id] from the Android calendar storage
-         * @throws IllegalArgumentException if event has not been saved yet
-         * @throws FileNotFoundException if there's no event with [id] in the calendar storage
-         * @throws RemoteException on calendar provider errors
-         */
+    private var _event: Event? = null
+
+    /**
+     * Returns the full event data, either from [event] or, if [event] is null, by reading event
+     * number [id] from the Android calendar storage
+     * @throws IllegalArgumentException if event has not been saved yet
+     * @throws FileNotFoundException if there's no event with [id] in the calendar storage
+     * @throws RemoteException on calendar provider errors
+     */
+    val event: Event?
         get() {
```

Comment:
I think we can also add a `private set()` that sets the `_event`. Then we can use the private setter in the other methods (constructor, maybe somewhere else, …) and they don't have to know whether they shall write to `_event` or directly to `event`.